### ISSUE TITLE
High Quality images for event gallery  that relates our theme

### ISSUE THREAD

**Issue by Surajbokde** [Surajbokde/Nature_Beauty#4]
![image](https://user-images.githubusercontent.com/66197682/100541095-95bace00-3267-11eb-84fe-0372324dc867.png)

change the event gallery images to nature and nature awareness images

**Comment by Suyash642** [comment]
Hello, Please Assign this issue to me.

**Comment by Surajbokde** [comment]
@Suyash642  i will assign this issue to you.
you have to replace all images with high quality images related to nature and nature awareness .
the path of all images are given in img/  folder.
and make PR after 6th dec

**Comment by Surajbokde** [comment]
@Suyash642 what is ur progress

**Comment by Suyash642** [comment]
I have replaced the entire gallery with new images which is mostly of  nature awareness campaign and nature. But due to different size they are not display in proper way.  Should I take all images of  same size?

The Gallery looks like this :-

![image](https://user-images.githubusercontent.com/58839369/101678756-68023000-3a84-11eb-8ff7-ecfa43544214.png)

I searched for it but didn't get satisfactory ans. Tell me how can I display them properly ?
